content/200-concepts/100-components/02-prisma-client/058-transactions.mdx

```diff
@@ -272,16 +275,83 @@ transaction functions. We recommend you get in and out as quick as possible!
 
 </Admonition>
 
-## Limitations
+#### Transaction isolation level
 
-### Transaction Isolation Level
+<Admonition type="info">
 
-Transaction isolation level is not currently configurable at a Prisma level and is not explicitly set by Prisma.
+This feature is not available on MongoDB, as MongoDB does not support isolation levels.
+
+</Admonition>
```

Comment:
Off topic: @zachtil I think we could have a specific "MongoDB" (or for any database) admonition with the appropriate logo as icon for such things. Maybe in a positive and "negative" variant, could be nice.

Reply:
Added it as a suggestion to the issue